command/tests/sample_program_websocket_test_inf/test_websocket_infinite_program.go

```diff
@@ -0,0 +1,10 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	fmt.Println("Print")
```

Comment:
I thought this was supposed to go into an infinite loop? What happens if the program doesn't quit by itself?